src/compojure_throttle/core.clj

```diff
@@ -61,18 +70,19 @@
   (:remote-addr req))
 
 (defn throttle
-"Throttle incoming connections from a given source. By default this is based on IP.
-
-Optionally takes a second argument which is a function used to lookup the 'token'
-that determines whether or not the request is unique. For example a function that
-returns a user token to limit by user id rather than ip. This function should accept
-the request as its single argument"
-([finder handler]
+  "Throttle incoming connections from a given source. By default this is based on IP.
+  
+  Optionally takes a second argument which is a function used to lookup the 'token'
+  that determines whether or not the request is unique. For example a function that
+  returns a user token to limit by user id rather than ip. This function should accept
+  the request as its single argument"
```

Comment:
I think it's worth adding an extra paragraph here to describe the behaviour changes from the `lax-ip` stuff.

Probably put it in between the two existing paragraphs, so it's like:

```clojure
"Throttle incoming connections from a given source. By default this is based on IP.

Throttling is controlled by both :service-compojure-throttle-enabled and 
:service-compojure-throttle-lax-ips. If
 :service-compojure-throttle-enabled is false, throttling will still happen 
to any IP not covered by :service-compojure-throttle-lax-ips.

Optionally takes a second argument which is ... "
```

Also, might be good to document how to disable throttling for multiple IPs at once - like, if they don't form a reasonable subnet with each other.

Reply:
>Also, might be good to document how to disable throttling for multiple IPs at once - like, if they don't form a reasonable subnet with each other.

uh... you can't. I was told a subnet would be sufficient (+ it def is for our internal needs, which was the primary concern of mine here)

Reply:
Ah, so it's just for our servers talking to each other. Not having special permissions for random other servers does simplify things.

How do we guarantee we're always in the same subnet? Part of being in a single VPC in AWS, I guess?

Reply:
Joey confirmed 1 subnet should be sufficient for our needs. Worst case, I actually only need one ip unthrottled.